src/backend/requirements.txt

```diff
@@ -1,9 +1,55 @@
+annotated-types==0.7.0
+anyio==4.6.2.post1
```

Comment:
I assume this is made using `pip freeze`. The issue is that it dumps all the packages in the environment, even the ones **NOT** needed for the project.

Reply:
yes, it is reason why there are a lot of unused packages, I will fix it later.